CHANGELOG.md

```diff
@@ -1,10 +1,11 @@
 5.0.3 Release notes (2020-4-01)
 =============================================================
 ### Enhancements
-* None.
+* None
 
 ### Fixed
-* Fixes Realm Object properties not working when accessed, returning undefined on React Native for Android in Realm JS v5.0.2  ([#2763](https://github.com/realm/realm-js/issues/2763))
+* Realm Object properties not working when accessed, returning undefined on React Native for Android in Realm JS v5.0.2  ([#2763](https://github.com/realm/realm-js/issues/2763))
+* Using of Realm Object instance methods obj.keys() and obj.entries() in TypeScript ([#2778](https://github.com/realm/realm-js/issues/2778))
```

Comment:
Can be better describe what didn't work? "when <this> then <this> happened."
E.g. copying the error description from the issue seems more descriptive?
"Using obj.keys() and obj.entries() caused TypeScript error "Property 'keys' does not exist on type 'Object'."

Reply:
👍